traffic_ops/app/lib/MojoPlugins/Health.pm

```diff
@@ -159,6 +159,9 @@ sub register {
 				cachegroups  => [],
 			};
 
+			$self->app->log->debug("get_cache_health() - " . 
```

Comment:
At first,  it would seem good practice,  but over time more of these would end up in the code.  If placed in a time-critical part of the code,  it could end up affecting performance.   We avoid this practice.